sass/themes/okc/overrides.scss

```diff
@@ -1,282 +1,366 @@
 /* General overrides */
 h1, .h1 {
-	font-size: $font-size-xlarge;
-	font-weight: $font-weight-bold;
+  font-size: $font-size-xlarge;
+  font-weight: $font-weight-bold;
 }
 
 h2, .h2 {
-	font-size: $font-size-large;
-	font-weight: $font-weight-bolder;
-	letter-spacing: initial;
+  font-size: $font-size-large;
```

Comment:
Having all these whitespace changes as part of c8de94f3e22079b5782066a347d83e618f134cc5 will make blaming & diffing very hard. I recommend doing a "`overrides.scss`: adapt indentation" commit before.